content/pages/changelog/index.md

```diff
@@ -10,6 +10,23 @@ rank: 0
 For historical changelog entries, please see
 [2018 Release Notes](page://changelog/2018).
 
+## 2019-11-25 / Marker-based Pagination Support for Trash API
```

Comment:
Can you pull in the current updates to the changelog and merge please? There is a conflict with the launch of the Elements 11 changelog entry.

Reply:
Done